dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKUtilities.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2022, 2023 IBM Corporation and others.
+ * Copyright (c) 2022, 2025 IBM Corporation and others.
```

Comment:
Copyright date should be 2024 - the date when you made and published (i.e. pushed somewhere public) the last non-trivial updates, not the date you expect it to be merged into integration.